### DISCUSSION TITLE
Any plans to make this library available throught React-native-web?

### DISCUSSION THREAD

**Discussion by Jose4gg** [margelo/react-native-filament#264] [Q&A]
Is there any ongoing development or future plan to support React-native-web for this library? I'm interested in using this library in a web project that shares code between web and mobile platforms, and having React-native-web support would be extremely helpful.

**Comment by hannojg** [comment]
Currently there aren't any. Contributions are more than welcome here 😊 👍

**Comment by zxffffffff** [comment]
If supported web, it will be a better solution than react-three-fiber 👍
